packages/make-styles-webpack-loader/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["es2015", "dom"],
```

Comment:
Not super important but I don't think you need `dom` here.

Reply:
Thanks, applied in 3add815 👍